### PR TITLE
chore: remove packages tests and bump versions

### PR DESCRIPTION
**Pull request by evenstensberg**
**What kind of change does this PR introduce?**

Updates dependencies for master branch and disables building and testing reduntant monorepo packages

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
N/A
**Summary**

Se first point

**Does this PR introduce a breaking change?**

No
**Other information**
N/A

### CODE REVIEW

**Review by evilebottnawi — COMMENTED** [review]
Why?

File: package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "format:js": "prettier-eslint ./bin/*.js ./bin/**/*.js ./test/**/*.js ./packages/**/**/*.js ./packages/**/*.js --write",
     "format:ts": "prettier-eslint ./packages/**/**/*.ts ./packages/**/*.ts ./packages/**/**/**/*.ts --write",
     "lint": "eslint \"./bin/*.js\" \"./bin/**/*.js\" \"./test/**/*.js\" \"packages/**/!(node_modules)/*.ts\"  \"packages/**/!(node_modules)/**/*.ts\"",
-    "pretest": "npm run build && npm run lint",
+    "pretest": "npm run lint",
```
Comment (evilebottnawi) [inline package.json:38]: Don't touch this

File: package.json [hunk 2]
```diff
@@ -96,8 +97,7 @@
   },
   "nyc": {
     "include": [
-      "bin/**.js",
-      "packages/**/*.js"
+      "bin/**.js"
```
Comment (evilebottnawi) [inline package.json:100]: Don't touch this, it should be done in PR where we store coverage after removing all tests between v3 and v4

File: package.json [hunk 3]
```diff
@@ -72,7 +72,8 @@
   },
   "jest": {
     "testPathIgnorePatterns": [
-      "/node_modules/"
+      "/node_modules/",
+      "/packages/"
```
Comment (evilebottnawi) [inline package.json:76]: https://github.com/webpack/webpack-cli/pull/1798/files#r487407963

**Comment by evenstensberg** [comment]
Monorepo fails to build, we don't need to build monorepo for a downstream

**Comment by evilebottnawi** [comment]
@evenstensberg Can you clarify?

**Comment by evenstensberg** [comment]
Try running npm test on master

**Review by rishabh3112 — CHANGES_REQUESTED** [review]
Is this PR required now?

**Comment by evilebottnawi** [comment]
I think we don't need it

**Pull request closed without merging**